### PR TITLE
Release notes 3.5.1

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### Proposed changes

Release notes for 3.5.1

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,22 @@ This page contains information about the new features, improvements, known issue
 
 **Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
 
+### Bug fixes and minor changes
+- Fixed a link to the policy that provides details on how Docker handles the uploaded diagnostics data. Fixes [docker/for-mac#5741](https://github.com/docker/for-mac/issues/5741)
+
+
```
Comment (mat007) [inline docker-for-mac/release-notes.md:31]: Double blank line

File: docker-for-windows/release-notes.md [hunk 2]
```diff
@@ -13,10 +13,26 @@ This page contains information about the new features, improvements, known issue
...
+
+**Dev Environments Preview**: Dev Environments enable you to seamlessly collaborate with your team members without moving between Git branches to get your code onto your team members' machines. When using Dev Environments, you can share your in-progress work with your team members in just one click, and without having to deal with any merge conflicts. For more information and for instructions on how to use Dev Environments, see [Development Environments Preview](../desktop/dev-environments.md).
+
+**Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
+
+### Bug fixes and minor changes
+- Fixed a bug where users could not install Docker Desktop when the temp folder contained dots. Fixes [docker/for-win#11514](https://github.com/docker/for-win/issues/11514)
+- Fixed a link to the policy that provides details on how Docker handles the uploaded diagnostics data. Fixes [docker/for-mac#5741](https://github.com/docker/for-mac/issues/5741)
+
+
```
Comment (mat007) [inline docker-for-windows/release-notes.md:31]: Ditto

File: docker-for-windows/release-notes.md [hunk 3]
```diff
@@ -13,10 +13,26 @@ This page contains information about the new features, improvements, known issue
...
+[Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn}
+
+### New
+
+**Dev Environments Preview**: Dev Environments enable you to seamlessly collaborate with your team members without moving between Git branches to get your code onto your team members' machines. When using Dev Environments, you can share your in-progress work with your team members in just one click, and without having to deal with any merge conflicts. For more information and for instructions on how to use Dev Environments, see [Development Environments Preview](../desktop/dev-environments.md).
+
+**Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
+
+### Bug fixes and minor changes
+- Fixed a bug where users could not install Docker Desktop when the temp folder contained dots. Fixes [docker/for-win#11514](https://github.com/docker/for-win/issues/11514)
```
Comment (mat007) [inline docker-for-windows/release-notes.md:28]: Should it be `the path to the temp folder …`?

Comment (lorenrh) [reply docker-for-windows/release-notes.md:28]: indeed!

File: docker-for-mac/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,21 @@ This page contains information about the new features, improvements, known issue
 
 **Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
 
+### Bug fixes and minor changes
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:28]: nit: add a blank line after the heading

File: docker-for-windows/release-notes.md [hunk 5]
```diff
@@ -13,10 +13,25 @@ This page contains information about the new features, improvements, known issue
...
+
+[Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn}
+
+### New
+
+**Dev Environments Preview**: Dev Environments enable you to seamlessly collaborate with your team members without moving between Git branches to get your code onto your team members' machines. When using Dev Environments, you can share your in-progress work with your team members in just one click, and without having to deal with any merge conflicts. For more information and for instructions on how to use Dev Environments, see [Development Environments Preview](../desktop/dev-environments.md).
+
+**Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md).
+
+### Bug fixes and minor changes
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md:27]: Add a blank line after the heading

Comment (lorenrh) [reply docker-for-windows/release-notes.md:27]: Thanks!

**Review by usha-mandya — APPROVED** [review]
LGTM

File: docker-for-mac/release-notes.md [hunk 6]
```diff
@@ -13,8 +13,8 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.5.0
-2021-06-23
+## Docker Desktop 3.5.1
+2021-06-24
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:17]:
```suggestion
2021-06-25
```

File: docker-for-windows/release-notes.md [hunk 7]
```diff
@@ -13,16 +13,32 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.5.1
+2021-06-24
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md:17]:
```suggestion
2021-06-25
```

**Review by usha-mandya — APPROVED** [review]
LGTM

**Review by StefanScherer — APPROVED** [review]
LGTM

File: docker-for-windows/release-notes.md [hunk 8]
```diff
@@ -13,16 +13,32 @@ This page contains information about the new features, improvements, known issue
...
+**Compose V2 beta**: Docker Desktop now includes the beta version of Compose V2, which supports the `docker compose` command as part of the Docker CLI. For more information, see [Compose V2 beta](../compose/cli-command.md). While `docker-compose` is still supported and maintained, Compose V2 implementation relies directly on the compose-go bindings which are maintained as part of the specification. The compose command in the Docker CLI supports most of the `docker-compose` commands and flags. It is expected to be a drop-in replacement for `docker-compose`. There are a few remaining flags that have yet to be implemented, see the [docker-compose compatibility list](../compose/cli-command-compatibility/) for more information about the flags that are supported in the new compose command. If you run into any problems with Compose V2, you can easily switch back to Compose v1 by either by making changes in Docker Desktop **Experimental** Settings, or by running the command `docker-compose disable-v2`. Let us know your feedback on the new ‘compose’ command by creating an issue in the [Compose-CLI](https://github.com/docker/compose-cli/issues) GitHub repository.
```
Comment (mat007) [inline docker-for-windows/release-notes.md:41]: Should we have that for 3.4.0 as well?

Comment (usha-mandya) [reply docker-for-windows/release-notes.md:41]: Oops. Just saw this. I can do a follow-up PR for this

**Pull request merged**